Tests/BenchmarkTests/BenchmarkReporterTests.swift

```diff
@@ -17,37 +17,35 @@ import XCTest
 @testable import Benchmark
 
 final class BenchmarkReporterTests: XCTestCase {
+    func testPlainTextReporter() throws {
+        let results: [BenchmarkResult] = [
+            BenchmarkResult(benchmarkName: "fast", suiteName: "My Suite", measurements: [1_000, 2_000]),
+            BenchmarkResult(benchmarkName: "slow", suiteName: "My Suite", measurements: [1_000_000, 2_000_000])
 
-    func testPaddingLeft() throws {
-        let testText = "testText"
-        XCTAssertEqual(testText.leftPadding(toLength: testText.count, withPad:" "), "testText")
-        XCTAssertEqual(testText.leftPadding(toLength: testText.count + 3, withPad:" "), "   testText")
-        XCTAssertEqual(testText.leftPadding(toLength: testText.count - 1, withPad:" "), "Triggar fatalError")
-    }
+        ]
+
+        let output = MockTextOutputStream()
+        var reporter = PlainTextReporter(to: output)
+
+        reporter.report(results: results)
 
-    func testPaddingEachCell() throws {
-        let dummyName = ["00", "10", "20"]
-        let dummyTime = ["01", "11", "21"]
-        let dummyStd = ["02", "12", "22"]
-        let dummyIterations = ["03", "13", "23"]
-        let columuns = [dummyName, dummyTime, dummyStd, dummyIterations]
-
-        for index in 0..<dummyName.count {
-            for columnIndex in 0..<columuns.count {
-                let cell = columuns[columnIndex][index]
-                let paddedCell = paddingEachCell(cell: cell,
-                    index: index, columnIndex: columnIndex, length: cell.count + 1)
-                if index != 0 && columnIndex == 1 {
-                    XCTAssertEqual(paddedCell, " \(index)\(columnIndex)")
-                } else {
-                    XCTAssertEqual(paddedCell, "\(index)\(columnIndex) ")
-                }
-            }
+        let expected = #"""
+        name                   time std        iterations
```

Comment:
`time` and `std` column names seems to be inconsistently aligned. I would suggest aligning all column names on the left for consistency.

Reply:
This is fixed by 242d42f6a95090c639d1ee6dd785cf709b22ad54